lib/itamae/plugin/recipe/rbenv/dependency.rb

```diff
@@ -5,8 +5,12 @@
   package 'build-essential'
   package 'libffi-dev'
   package 'libgdbm-dev'
-  if node[:platform] == 'ubuntu' && node[:platform_version] >= '18.04'
-    package 'libgdbm5'
+  if node[:platform] == 'ubuntu'
```

Comment:
I think this also impacts libreadline-dev package name, but xenial (16.04) seems to use libreadline6-dev?

Reply:
Ah, Correctly. How about the https://github.com/k0kubun/itamae-plugin-recipe-rbenv/pull/30/commits/6e16c21092d915b26912a8bfcc49b0dabfef59e7?